experimental/formatted_string/src/formatted_string.rs

```diff
@@ -3,16 +3,32 @@
 // (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
 
 use crate::FormattedStringBuilderError;
+use alloc::boxed::Box;
+use alloc::string::String;
+use alloc::{vec, vec::Vec};
+use core::str::from_utf8_unchecked;
```

Comment:
nit: typical style for calling free functions is to import their module (`core::str`) and invoke them with the module name (`str::from_utf8_unchecked()`)